templates/federated_reporting/psql_wrapper.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# run an arbitrary psql command, either -f or -c
+# expectation is that a line will be output with exit_code=<n>
+# this exit code <n> will then be used to exit
+# This design enables cfengine policy to run a commands promise
+# that can be kept, repaired or failed depending on an exit
+# code from psql since psql can't get different codes from scripts.
+
+if [ $# -ne 2 ]
+then
+    echo "Usage: $0 [DBNAME] [SQL STRING]"
+    exit 2
+fi
+
+TMP=$(mktemp)
+TMP2=$(mktemp)
+OUT=$(psql --quiet --tuples-only --no-align --no-psqlrc "$1" -c "$2" 2> $TMP ; echo $? > $TMP2)
+ERR=$(<$TMP)
+
+EXIT_CODE=$(echo $TMP | awk -F= '{ if ( /exit_code/ ) print $2}')
+RETURN_CODE=$(<$TMP2)
+
+rm $TMP $TMP2
+
+if (( $RETURN_CODE != 0 )); then
```

Comment:
Please stick to one way of shell syntax and code style. The two `if` statements in this file look totally different while they are doing the same.

Reply:
fixed